actor/builtin/miner/miner_test.go

```diff
@@ -118,26 +117,54 @@ func TestAskFunctions(t *testing.T) {
 	assert.Len(t, askids, 2)
 }
 
-func TestGetKey(t *testing.T) {
+func TestGetWorker(t *testing.T) {
 	tf.UnitTest(t)
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
 	st, vms := core.CreateStorages(ctx, t)
 
-	signature := []byte("my public key")
-	minerAddr := createTestMiner(t, st, vms, address.TestAddress, signature, th.RequireRandomPeerID(t))
+	minerAddr := createTestMiner(t, st, vms, address.TestAddress, th.RequireRandomPeerID(t))
 
 	// retrieve key
-	result := callQueryMethodSuccess("getKey", ctx, t, st, vms, address.TestAddress, minerAddr)
-	assert.Equal(t, result[0], signature)
+	result := callQueryMethodSuccess("getWorker", ctx, t, st, vms, address.TestAddress, minerAddr)
```

Comment:
This isn't testing that it retrieves the right address since it's impossible to distinguish between the owner and worker address at creation.

Reply:
This is true. The next PR adds the ability to change the address. I've already written that test.